set internalRangeLookup INCONSISTENT in DistSender

tbg opened the pull request:
and (to avoid confusion) do not do it in the r.InternalRangeLookup
itself. The reason for this change is that we will soon take the
consistency level into account for routing calls to replicas.

related to #543

spencerkimball commented on storage/range.go:
```diff
@@ -1101,7 +1101,9 @@ func (r *Range) InternalRangeLookup(batch engine.Engine, args *proto.InternalRan
 	metaPrefix := proto.Key(args.Key[:len(engine.KeyMeta1Prefix)])
 	nextKey := proto.Key(args.Key).Next()
-	kvs, err := engine.MVCCScan(batch, nextKey, metaPrefix.PrefixEnd(), rangeCount, args.Timestamp, false, args.Txn)
+	// This will always be false, at least when called from the DistSender.
```
s/This will always be/Always/

spencerkimball commented:
LGTM

tbg committed:
set internalRangeLookup INCONSISTENT in DistSender

and (to avoid confusion) do not do it in the r.InternalRangeLookup
itself. The reason for this change is that we will soon take the
consistency level into account for routing calls to replicas.
